front/tests/pages/operational-studies/operational-studies-page.ts

```diff
@@ -89,28 +89,20 @@ class OperationalStudiesPage extends CommonPage {
   }
 
   // Click on the button to add a scenario train.
-  async clickOnAddTrainButton() {
+  async openTrainScheduleForm() {
```

Comment:
Since we do not handle train schedule only
```suggestion
  async openTimetableItemForm() {
```